.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Short description explaining the high-level reason for the new issue.
+
+## Current behavior
+
+-
+
+## Expected behavior
+
+-
+
+## Steps to replicate behavior
+
+1.
+
+
+## Screenshots
+
+__Current__
+
+
+__Expected__
```

Comment:
My concern is content team members or members of the public who fill out the issue. I don't want to make the bar to contribute any higher than it needs to be. A note about it being optional sounds great.